src/actions/LoginThunkActions.ts

```diff
@@ -73,14 +74,27 @@ const LoginThunkActions = {
   },
   logout: () => {
     return async (dispatch: KialiDispatch) => {
-      try {
-        const response = await API.logout();
+      if (authenticationConfig.strategy === AuthStrategy.openshift && authenticationConfig.logoutEndpoint) {
+        axios
+          .post(authenticationConfig.logoutEndpoint)
+          .then(() => {
+            if (authenticationConfig.logoutRedirect) {
+              window.location.assign(authenticationConfig.logoutRedirect);
```

Comment:
I don't think this will work. The logout endpoint will only accept a request coming in from a POST, it wont accept any other method. This means we cannot use xmlhttprequests/ajax and we cannot use window.location.assign (which will result in a GET).

The logout endpoint will only work if you access it via a POST like what a form submit does.

Reply:
Mmm but this is a axios POST, I'll check it tomorrow the status code.